Updated README file

pekboy opened the pull request:
1. Updated build instruction
2. Added quickstart section

evance-magnet committed:
Update sample apps to remove deprecated APIs.

evance-magnet committed:
Additional changes for removing deprecated APIs in SoapBox

tsk70 committed:
Initial commit for Smart Shopper

vampserv committed:
change sdk version

tsk70 committed:
changes to trim the sale price doller sign

tsk70 committed:
Merge branch 'develop' of https://github.com/magnetsystems/message-samples-android into develop

evance-magnet committed:
Initial commit of WRU app.

From bitbucket, 7e857200520965569da093187de89cce55b4e4c1

tsk70 committed:
removing the unused variable

priteshshah1983 committed:
Added README

priteshshah1983 committed:
Added walkthrough

evance-magnet commented:
Something looks wrong here...   There are quite a few files modified.   What's the actual change?

evance-magnet committed:
Move the loadLastLocations() earlier to prepopulate locations.

Call requestUpdates() after successful login.

vampserv committed:
remove amazon s3 key, add instructions for adding keys

jimxj committed:
Update README.md

jimxj committed:
Update README.md

jimxj committed:
Update README.md

evance-magnet committed:
Add connection status colors.

magnetci committed:
Changing version from 1.6.0-SNAPSHOT to 1.10.0-SNAPSHOT

evance-magnet committed:
Fix bug where connect would not be reattempted if initial login failed.

Also fixed a but where saveLastLocation and loadLastLocations were not
working properly because of upper vs. lower topic name.

evance-magnet committed:
Google Play Services is no longer included in SDK 1.10+ so adding where needed.

The newest version of the Magnet Message Android SDK will no longer declare
Google Play Services base and gcm as a dependency.  In order to enable this functionality
it needs to be declared as a depedency in the app's build.gradle file and
the meta-data element needs to be added to the AndroidManifest.xml

Commit:
Merge branch 'develop' of https://github.com/magnetsystems/message-samples-android into develop

Commit:
Merge branch 'multicast_msg' into develop

Conflicts:
	quickstart/app/src/main/java/com/magnet/demo/mmx/starter/MyApplication.java
